src/EFCore.PG.NodaTime/Query/ExpressionTranslators/Internal/NpgsqlNodaTimeMemberTranslatorPlugin.cs

```diff
@@ -76,9 +76,49 @@ public NpgsqlNodaTimeMemberTranslator(ISqlExpressionFactory sqlExpressionFactory
                 return TranslateDateTime(instance, member, returnType);
             }
 
+            if (instance is not null && declaringType == typeof(Duration))
```

Comment:
nit: can extract out `instance is not null` check for this and the above code blocks.

Reply:
done